TestForSignUp/InputTextGenerator.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace TestForSignUp
+{
+    internal class InputGenerator
```

Comment:
Why do we need internal here?

Reply:
Because this class is used only in this project